packages/react-native/ReactCommon/react/renderer/core/RawPropsParser.h

```diff
@@ -26,8 +26,11 @@ class RawPropsParser final {
   /*
    * Default constructor.
    * To be used by `ConcreteComponentDescriptor` only.
+   * If `useRawPropsJsiValue` is `true`, the parser will use `jsi::Value`
+   * directly for RawValues instead of converting them to `folly::dynamic`.
    */
-  RawPropsParser() = default;
+  RawPropsParser(bool useRawPropsJsiValue = false)
```

Comment:
This way we could remove the additional check and just read this once.

```suggestion
  RawPropsParser(bool useRawPropsJsiValue = ReactNativeFeatureFlags::useRawPropsJsiValue())
```